homeassistant/components/sensor/rainbird.py

```diff
@@ -0,0 +1,81 @@
+"""
+Support for Rain Bird Irrigation system LNK WiFi Module.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/sensor.rainbird/
+"""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.rainbird import DATA_RAINBIRD
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import CONF_MONITORED_CONDITIONS
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.helpers.entity import Entity
+
+DEPENDENCIES = ['rainbird']
+
+_LOGGER = logging.getLogger(__name__)
+
+# sensor_type [ description, unit, icon ]
+SENSOR_TYPES = {
+    'rainsensor': ['Rainsensor', None, 'water']
+}
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_MONITORED_CONDITIONS, default=list(SENSOR_TYPES)):
```

Comment:
Can we make this optional, since it has a default?

Reply:
Correct